loops_lab.js

```diff
@@ -0,0 +1,203 @@
+// <!-- # Loops Lab
+//
+// To complete this lab, do the following:
+//
+// 1. Fork this repo
+// 2. Clone **your** forked repo
+// 3. Create a file called `loops_lab.js` inside your cloned repo
+// 4. Copy the description of the exercise as a comment
+// 5. Underneath the commented description write your code.
+// 6. Ensure your solutions work properly!
+// 7. Open PR against this repo.
+//
+// ---
+//
+// ## Exercises
+//
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1. -->
+//
+//   let num = 23;
+// while (num >= 1) {
+// console.log(num);
+// num -= 1;
+// }
+// console.log("--------------");
+// for (let num = 23; num >= 1; num -= 1) {
+// console.log(num);
+// }
+//
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+//    For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+//
+
+//
+// 3. Write a `while` loop and a `for` loop that takes the variable "x" and iterates over all numbers from 0 to "x".
+//    For each iteration of the loop, it will multiply the number of "x" by 9 and log the result (e.g. "2 \* 9 = 18").
+//    _Bonus_ think of another way to solve it.
+// for (let num = 0; num < 36; num += 1) {
+//   console.log(num + ' * 9 =' + num * 9)
+// }
+// console.log('-----------------');
+
+//
+//    <details>
+//    <summary>
+//    Hint
+//    </summary>
+//    Find the final number and increment the loop by 9.
+//    </details>
+//
+// 4. Write code that prints/logs all the numbers that end in a 5 from 1 to 100, **exclusive**.
```

Comment:
Since it's asking from 1 to 100, what would happen if we change x = 1? When I tested it, it gave me 1, 11, 21, etc. Would there be any way for us to use if statements and modulo to solve this? Think about it.